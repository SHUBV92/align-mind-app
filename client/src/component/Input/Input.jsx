import React from "react";

const Input = ({ label, type, placeholder, name, onChange, value }) => {
  return (
    <div>
      <label>{label}</label>
      <input
        type={type}
        placeholder={placeholder}
        name={name}
        onChange={onChange}
        value={value}
      />
    </div>
  );
};

export default Input;
