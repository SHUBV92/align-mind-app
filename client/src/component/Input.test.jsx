import React from "react";
import { render } from "@testing-library/react";
import Input from "./Input";

it("should take snapshot", () => {
  const { asFragment } = render(<Input />);

  expect(asFragment(<Input />)).toMatchSnapshot();
});
