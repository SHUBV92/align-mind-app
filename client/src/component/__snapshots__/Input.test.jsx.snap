// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take snapshot 1`] = `
<DocumentFragment>
  <div>
    <form>
      <label />
      <input />
    </form>
  </div>
</DocumentFragment>
`;
